manage-data/data-store/data-streams/tsds-ingest-otlp.md

```diff
@@ -0,0 +1,98 @@
+    sending_queue:
+      enabled: true
+      sizer: requests
+      queue_size: 5000
+      block_on_overflow: true
+      batch:
+        flush_timeout: 5s
+        sizer: bytes
+        min_size: 2_000_000
+        max_size: 5_000_000
+    auth:
+      authenticator: basicauth/elasticsearch
+```
+
+The supported options for `compression` are `gzip` (default value of the `OTLP/HTTP` exporter) and `none`.
+
+% TODO we might actually also support snappy and zstd, test and update accordingly)
+
+To track metrics in your custom application,
+use the [OpenTelemetry language SDK](https://opentelemetry.io/docs/getting-started/dev/) of your choice.
+
+:::{note} 
+Only `encoding: proto` is supported, which the `OTLP/HTTP` exporter uses by default.
+:::
+
+## Best practices
+
+Don't send metrics from applications directly to the {{es}} OTLP endpoint, especially if there are many individual applications that periodically send a small amount of metrics. Instead, send data to an OpenTelemetry Collector first. This helps with handling many connections, and with creating bigger batches to improve ingestion performance. 
+
+On {{ecloud}}, use the [{{motlp}}](opentelemetry:/reference/motlp.md) and for self-managed use cases, you can use the [Elastic Distribution of OpenTelemetry Collector](elastic-agent:/reference/edot-collector/index.md).
+
+For more details on the recommended way to set up OpenTelemetry-based data ingestion, refer to the [EDOT reference architecture](opentelemetry:/reference/architecture/index.md).
+
+## Send data to different data streams
+
+By default, metrics are ingested into the `metrics-generic.otel-default` data stream. You can influence the target data stream by setting specific attributes on your data:
+
+- `data_stream.dataset` or `data_stream.namespace` in attributes, with the following order of precedence: data point attribute -> scope attribute -> resource attribute
+- Otherwise, if the scope name contains `/receiver/<somereceiver>`, `data_stream.dataset` is set to the receiver name.
+- Otherwise, `data_stream.dataset` falls back to `generic` and `data_stream.namespace` falls back to `default`.
+
+The target data stream name is constructed as `metrics-${data_stream.dataset}.otel-${data_stream.namespace}`.
+
+## Limitations
+
+* Only the OTLP metrics endpoint (`/_otlp/v1/metrics`) is supported.
+  To ingest logs, traces, and profiles, use a distribution of the OpenTelemetry Collector that includes the [{{es}} exporter](opentelemetry:/reference/edot-collector/components/elasticsearchexporter.md),
+  such as the [Elastic Distribution of OpenTelemetry (EDOT) Collector](opentelemetry:/reference/edot-collector/index.md).
+* Histograms are only supported in delta temporality. Set the temporality preference to delta in your SDKs, or use the [`cumulativetodelta` processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) to avoid cumulative histograms to be dropped.
```

Comment:
should we add this to the example config?

Reply:
I feel like that's beyond the scope for the Elasticsearch docs. We may want to add this to the OTel docs and link to it.